README.md

```diff
@@ -67,12 +67,15 @@ Myne doesn't contain any ads and doesn't sell your data.
 The development of the app is financed by individual user contributions, such as you purchasing the app via Google Play or becoming a sponsor on Github ❤️
 
 Become a [Sponser](https://github.com/sponsors/starry-shivam) on Github | Purchase it on [Google Play](https://play.google.com/store/apps/details?id=com.starry.myne)
-
```

Comment:
Add this space back because if the divider touches the text, the bottom line "Become a [Sponsor](https://github.com/sponsors/starry-shivam) on Github | Purchase it on [Google Play](https://play.google.com/store/apps/details?id=com.starry.myne)" will become as big as header texts since the divider won't get applied.